### PR TITLE
fix(usage plan): new list price table

### PR DESCRIPTION
**Pull request by zuluecho9**

### CODE REVIEW

**Review by homelessbirds — APPROVED** [review]

File: src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx [hunk 1]
```diff
@@ -138,10 +138,174 @@ The Order and Usage Plan may contain defined terms that are denoted by capitaliz
...
+              USD $0.05
+            </td>
+          </tr>
+
+        </tbody>
+      </table>
+
+Notes:
+<sup>1</sup> Your Order and/or invoice will include both the Product name and applicable edition for your subscription.
+<sup>2</sup> List Prices are promotional and subject to change, and the prices above are for non-Pay As You Go plans and Orders with annual billing terms. List Prices for the New Relic Pro Full Platform Users, New Relic Enterprise Full Platform Users, and New Relic - Core Users shall be an additional 12% for non-Pay As You Go plans with monthly billing terms. List Prices for New Relic Pro Full Platform Users and New Relic Enterprise Full Platform Users shall be an additional 20% for Pay As You Go plans. All amounts are in USD.
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx:304]: @zuluecho9 I stumbled over the "non-" constructions. Could you change it to this: non-Pay-As-You-Go plans? Also, do you need to capitalize "Order" and "Prices"?

File: src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx [hunk 2]
```diff
@@ -138,10 +138,174 @@ The Order and Usage Plan may contain defined terms that are denoted by capitaliz
...
+<sup>3</sup> You may only utilize one of the following user editions at any given time: New Relic - Standard Full Platform Users, New Relic - Pro Full Platform Users, or New Relic - Enterprise Full Platform Users. You may only utilize one data option at any given time: New Relic - Data or New Relic - Data Plus.
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx:305]: @zuluecho9 what does this mean? Are you saying that companies can't have a mix of different types of users?

Also, I'd insert a carriage return between this footnote and #2 to keep them separate.

**Review by rhetoric101 — APPROVED** [review]
@zuluecho9 I inserted some suggestions below.

File: src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx [hunk 3]
```diff
@@ -138,10 +138,174 @@ The Order and Usage Plan may contain defined terms that are denoted by capitaliz
...
+            <td>
+              USD $0.05
+            </td>
+          </tr>
+
+        </tbody>
+      </table>
+
+Notes:
+<sup>1</sup> Your Order and/or invoice will include both the Product name and applicable edition for your subscription.
```
[the remark this answers is not in the document]
Comment (zuluecho9) [reply src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx:303]: those are legal requirements, some of them they capitalize in the license docs due to them being terms in order forms and such

File: src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx:305]: that's just saying they can't have more than one edition at same time (they bill by users though)

**Pull request merged**